elements/rh-spinner/rh-spinner.ts

```diff
@@ -59,6 +43,22 @@ export class RhSpinner extends BaseSpinner {
       <slot></slot>
     `;
   }
+
+  // START hack for removal of contextProvider. delete for version 2.0
+  /**
+   * @deprecated Use Color context instead. See https://ux.redhat.com/foundations/color/context/
```

Comment:
I can't find this in the deploy preview. Does this need to be created in a separate issue?

Reply:
It looks like that page doesn't exist yet, so we can add it after or as part of #1298.  We will want an issue created for it though for tracking

Reply:
@brianferry, adding it as part of #1298 makes sense. I'm ok with this merging in the meantime.

Reply:
yeah exactly @brianferry, #1298 is the pr for that. I'm hoping that's enough to merge this PR in, and if the URL changes in #1298, we can update the reference here in rh-spinner.ts over there in that pr